### PR TITLE
Add Blackbox monitor server to Liquidator bot

### PR DESCRIPTION
**Pull request by nicholaspai**
This adds a blackbox monitor server to the Liquidator. Follow on PR's to add this to other bots. We need to be confident that this is compatible with GCP health check infra.

Signed-off-by: Nick Pai <npai.nyc@gmail.com>

### CODE REVIEW

**Comment by nicholaspai** [comment]
@mrice32 @chrismaree Would love to get your feedback before I do more work.

**Comment by chrismaree** [comment]
This looks great! I think it'll do exactly what we need for the GCP health checks

File: common/ServerUtils.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  // Define routes.
+  app.get("/", (req, res) =>
+    res.status(200).json({
+      message: "Bot is up"
+    })
+  );
+  //
+  // source: https://expressjs.com/en/4x/api.html#app.listen
+  const server = app.listen(port);
```
[the remark this answers is not in the document]
Comment (nicholaspai) [reply common/ServerUtils.js:25]: The port's already optional checkout the comments. The default is a random unused port, which is great for testing. In production, we should specify port numbers to make sure we don't re-use any

[the remark this answers is not in the document]
Comment (mrice32) [reply common/ServerUtils.js:25]: Agreed. I like the way this is handled in express.

**Pull request merged**